gtsam/hybrid/HybridGaussianConditional.cpp

```diff
@@ -235,7 +237,7 @@ std::shared_ptr<HybridGaussianFactor> HybridGaussianConditional::likelihood(
           c << std::sqrt(2.0 * Cgm_Kgcm);
           auto constantFactor = std::make_shared<JacobianFactor>(c);
           gfg.push_back(constantFactor);
-          return std::make_shared<JacobianFactor>(gfg);
+          return {std::make_shared<JacobianFactor>(gfg), 0.0};
```

Comment:
Ok, I don’t get it: if we now have the ability to add a constant, why don’t we? If you want to do the hiding, do it in HGF?

Reply:
I was going to do this in a possible next PR, but done now. :)